### PR TITLE
Fix: Key operations table show NaN if no value

### PR DESCRIPTION
**Pull request by ksorv**
image

As seen in the screenshot, for my initial setup of the repo, I think some dummy data comes..

But there is no check for what happens if `value` in the table operations is undefined, leading to `NaN`. 

Fixed by simply using 0 of value is falsy.

Please check and let know if its needed.

### CODE REVIEW

File: frontend/src/container/MetricsApplication/TopOperationsTable.tsx [hunk 1]
```diff
@@ -93,7 +93,7 @@ function TopOperationsTable(props: TopOperationsTableProps): JSX.Element {
 			sorter: (a: TopOperationList, b: TopOperationList): number =>
 				a.errorCount - b.errorCount,
 			render: (value: number, record: TopOperationList): string =>
```
Comment (palashgdev) [inline frontend/src/container/MetricsApplication/TopOperationsTable.tsx:95]: can you please update the type also would be good

**Review by palashgdev — CHANGES_REQUESTED** [review]

File: frontend/src/container/MetricsApplication/TopOperationsTable.tsx (hunk 1 above)
Comment (ksorv) [reply frontend/src/container/MetricsApplication/TopOperationsTable.tsx:95]: yes, sorry missed those. Will add them right away.

Comment (ksorv) [reply frontend/src/container/MetricsApplication/TopOperationsTable.tsx:95]: @palashgdev Sorry for being late.

I had a question, Should I change the `TopOperationList` type itself or the `render` fn types are enough? 

I wanna do it in `TopOperationList` lemme know I'll fix it asap. Thanks.

Comment (palashgdev) [reply frontend/src/container/MetricsApplication/TopOperationsTable.tsx:95]: Hi @ksorv 
updating `TopOperationList` would make sense here

**Pull request closed without merging**